spec/libsass/bourbon.hrx

```diff
@@ -1,3 +1,11 @@
+<===> options.yml
+
+:todo:
+# This file starts failing when split, sass compiler seems to struggle when
+# `input.scss` is a real file with `@import "a/b/c"` and `a/b` is actually an
+# HRX file containing `c` in its root.
+- lint-hrx
+
```

Comment:
For some reason this file could not be split, it started throwing errors, so I figured I should leave it as-is for now which means this tool now has a way to skip HRX that shouldn't get split for whatever reason